roles/openshift_master/tasks/main.yml

```diff
@@ -23,7 +23,7 @@
       api_port: "{{ openshift_master_api_port | default(None) }}"
       api_url: "{{ openshift_master_api_url | default(None) }}"
       api_use_ssl: "{{ openshift_master_api_use_ssl | default(None) }}"
-      public_api_url: "{{ openshift_master_public_api_url | default(None) }}"
+      public_api_url: "{{ openshift_master_public_api_url | default('https://' ~ openshift.common.public_ip ~ ':8443') }}"
```

Comment:
It was a correction of this file at the revision 6fb39711a57bdad8f91c88504c70376f85cbb850 : 
```
-      public_api_url: "{{ openshift_master_public_api_url | default(openshift.common.public_ip) }}"
+      public_api_url: "{{ openshift_master_public_api_url | default('https://' ~ openshift.common.public_ip ~ ':8443') }}"
```
I tried to know where the variable openshift_master_public_api_url is defined.
With a simple grep, it shows : 
```
playbooks/common/openshift-master/config.yml
29:          public_api_url: "{{ openshift_master_public_api_url | default(None) }}"

roles/openshift_master/tasks/main.yml
26:      public_api_url: "{{ openshift_master_public_api_url | default('https://' ~ openshift.common.public_ip ~ ':8443') }}"

roles/openshift_master/README.md
25:| openshift_master_public_api_url     | UNDEF                 | |

```
It seems this variable is never defined, so public_api_url was always defaulting to the public_ip, without https:// and port. So this is to make it work.

Currently it's defaulting to None. Is it working ?